api/CanvasRenderingContext2D.json

```diff
@@ -2123,6 +2123,53 @@
           }
         }
       },
+      "getContextAttributes": {
```

Comment:
Yep, looks like this isn't in Gecko or WebKit. They do have `getContextAttributes` for WebGL though.